### PR TITLE
Created a basic skeleton for microservices

### PR DESCRIPTION
**Pull request by rshanmugam-enquizit**
- Let's start using this as a starting point for long term services creation

### CODE REVIEW

File: etl-data-pipeline/observation/build.gradle
Comment (alog-enquizit) [inline etl-data-pipeline/observation/build.gradle:1]: This looks good! Let's make sure that each of the individual modules can be built independently and jar is produced for each of them.  Based on that we might have to update the docker config.

File: etl-data-pipeline/person/build.gradle
Comment (alog-enquizit) [inline etl-data-pipeline/person/build.gradle:1]: Person and Provider will have the same module. We might have to add one for the Investigation.
Basically:
1. Observation
2. Investigation
3. Person - (Patient, Provider)
4. Organization

**Review by alog-enquizit — APPROVED** [review]
lgtm 👍

**Comment by alog-enquizit** [comment]
Let's also test out the docker container creation before we merge.

**Comment by rseshadri99** [comment]
@alog-enquizit @rshanmugam-enquizit Could we move all micro services under a folder /java/.
/java
.. //each micro service project
/sql
../odse
..../procs

Something like tihs

**Comment by rshanmugam-enquizit** [comment]
> @alog-enquizit @rshanmugam-enquizit Could we move all micro services under a folder /java/. /java .. //each micro service project /sql ../odse ..../procs
> 
> Something like tihs

@rseshadri99 that is a non-standard procedure and is there a reason you would want to go that way?

**Comment by rseshadri99** [comment]
On the SQL side - it helps with deploying the scripts as needed w/o a java code being involved in a build. If we use an automated tool as well, the structure would be more dictated by the tool. 
The reason for java as a folder is in the future if there are python-based modules that need to be integrated with data pipeline  - it helps to have those structure

**Pull request merged**